de-lib/src/main/java/com/sencha/gxt/widget/core/client/grid/LiveGridCheckBoxSelectionModel.java

```diff
@@ -270,32 +261,16 @@ protected void onRemove(DiskResource model) {
 
     @Override
     protected void onRowClick(RowClickEvent event) {
```

Comment:
Do we need to override this,`onRowMouseDown`, or `onSelectChange` still if we're just calling super?